convert2rhel/checks.py

```diff
@@ -75,81 +91,79 @@ def perform_pre_ponr_checks():
 
 
 def check_convert2rhel_latest():
-    """Make sure that we are running the latest version of convert2rhel"""
-    gpg_path = os.path.join(utils.DATA_DIR, "gpg-keys", "RPM-GPG-KEY-redhat-release")
-    ssl_cert_path = os.path.join(utils.DATA_DIR, "redhat-uep.pem")
-    repo_content = (
-        "[convert2rhel]\n"
-        "name=Convert2RHEL Repository\n"
-        "baseurl=https://cdn.redhat.com/content/public/convert2rhel/$releasever/x86_64/os/\n"
-        "gpgcheck=1\n"
-        "enabled=1\n"
-        "sslcacert=%s\n"
-        "gpgkey=file://%s\n" % (ssl_cert_path, gpg_path)
-    )
+    """Make sure that we are running the latest downstream version of convert2rhel"""
+    logger.task("Prepare: Checking if this is the latest version of Convert2RHEL")
+
+    if not system_info.has_internet_access:
+        logger.warning("Skipping the check because no internet connection has been detected.")
+        return
 
     cmd = [
         "repoquery",
         "--releasever=%s" % system_info.version.major,
+        "--repofrom='convert2rhel,https://cdn.redhat.com/content/public/convert2rhel/$releasever/$basearch/os/'convert2rhel",
+        "convert2rhel",
```

Comment:
We now define the CDN URL both here and in `REPO_CONTENT`. I'd consolidate the two into one variable that's used in both

I see it was added in `3e162a01fdb2ab0246dddefd084bb5989a9c82f7`, but that commit message is hiding the changes in the commit. If you need to be concise with commit messages I'd suggest to mention changes and refactors first, then fixes, and lastly rewordings/fixing typos

Reply:
Thank you for this comment! I know that's an area where I could get better with! I will consolidate those two together and push that.